scripts/o3de/tests/unit_test_enable_gem.py

```diff
@@ -57,6 +57,31 @@
 }
 '''
 
+TEST_O3DE_MANIFEST_JSON_PAYLOAD = '''
+{
+    "o3de_manifest_name": "testuser",
+    "origin": "C:/Users/testuser/.o3de",
+    "default_engines_folder": "C:/Users/testuser/.o3de/Engines",
+    "default_projects_folder": "C:/Users/testuser/.o3de/Projects",
```

Comment:
These test do not run on Linux, but none of the paths themselves are used open files.
They are there to provide values for keys within the manifest files.
The tests are made in such a way to avoid accessing files on the actual filesystem.
For example the [unit_test_cmake.py](https://github.com/o3de/o3de/blob/development/scripts/o3de/tests/unit_test_cmake.py#L121-L135) uses io.StringIO for in memory file operations.

Reply:
While CTest do not have any of these test running on Linux, I decided to directly run this particular test fgile through pytest an found out that one of three test were failing.
So I made a fix and now the test will pass on all three host platforms.